CarInspection/src/WebCarInspection/WebCarInspection.csproj

```diff
@@ -5,13 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.7" />
```

Comment:
This package useless

Reply:
This package is used to transfer and send objects to json API

Reply:
Anyway, you should use clean HttpClient, or RestEase, for example.